### DISCUSSION TITLE
Replying to new language request

### DISCUSSION THREAD

**Discussion by brockfanning** [WeblateOrg/weblate#5892] [Q&A]
Hello all, I am probably missing something obvious, but here is my issue: I have components configured with an "Adding new translation" of "Contact maintainers". This is working fine, but in some cases I need to ask the requester a question. Is there any way to send a message and/or email to the requester of a new language? I suppose I could go into an arbitrary translation and ping them (using the @ symbol) in a comment. But I'm hopeful there is a cleaner solution.

Thank you!

**Comment by nijel** [comment]
The user e-mail used to be exposed in Reply-To, but it was removed for privacy reasons in ea11e617b0cdf2944840653fc5a69ffa237ccb8b.

**Reply by brockfanning** [reply]
Thanks @nijel that makes sense. Do you think that pinging them with @ in a comment on an arbitrary translation is the best way? I also considered posting an "announcement" and pinging them there, but that seems a lot more public.

**Reply by nijel** [reply]
In case that person contributed to your repository, you can also find out e-mail in Git history.

We really need to introduce way to contact users...

**Reply by brockfanning** [reply]
Understood, thanks very much for the info!
